Makefile

```diff
@@ -10,8 +12,15 @@ build: $(BIN)
 docker: Dockerfile clean
 	docker build . --force-rm -f Dockerfile -t indexer-reference-provider:$(shell git rev-parse --short HEAD)
 
+# Build index-provider executable.  Use a temporary cmd/go.mod to build using
+# code from the current project directory.
 $(BIN): vet test
-	go build -modfile $(CMD_SUBDIR)/go.mod -o $@ $(CMD_SUBDIR)/provider/*.go
+	@cp -f $(CMD_SUBDIR)/go.mod $(TMP_MODFILE)
+	@cp -f $(CMD_SUBDIR)/go.sum $(TMP_SUMFILE)
+	@go mod edit -modfile $(TMP_MODFILE) -replace github.com/filecoin-project/index-provider=./
```

Comment:
Seen this? https://github.com/filecoin-project/index-provider/pull/95#discussion_r750520378